### ISSUE TITLE
Option to disable (or customize) `className`

### ISSUE THREAD

**Issue by kentcdodds** [brophdawg11/remix-validity-state#2]
I style with Tailwind and would prefer to not have other classNames on my elements.

**Comment by kentcdodds** [comment]
Looks like I can override the className by passing it. Normally I would expect this to combine the className I provide with the built-in one. If I want to completely override it I can simply specify my own `className` prop after the object spread.

**Comment by brophdawg11** [comment]
Great idea!  

```jsx
// append
<input {...getInputAttrs({ className: "foo bar" })} />

// replace
<input {...getInputAttrs()} className={"foo bar"} />
```

**Comment by brophdawg11** [comment]
Should be available in `remix-validity-state@0.3.2` 👍 

Were you also suggesting a global flag to turn them off, via the `FormContext` maybe?  I can't thin of a great use case for that at the moment though so I held off for now

**Comment by kentcdodds** [comment]
Thanks!

Agreed. I don't think I personally need it disabled globally.
